### DISCUSSION TITLE
Switching from double blinded screening to single screening - adjusting dataset & starting new project in ASReview

### DISCUSSION THREAD

**Discussion by Morle2104** [asreview/asreview#1854] [Q&A]
Dear all, 

We are currently employing blinded independent screening in ASReview, having two seperate project files with the same dataset, the same classifier & data extraction technique, and the same prior knowledge. On the basis of good inter-rater agreement, we now want to proceed with single screening (i.e., both screeners sequentially screening in one dataset to save resources). 

I am wondering how to best merge the two datasets without losing information, given that some articles were only seen by one of the two screeners. 
My idea is to download the complete Excel files of both projects, identify the records that were only labelled by one of the screeners, and subsequently _manually_ add these labels to a merged dataset that features all screening decisions that were made until now. Thus, the merged dataset includes all screened records, including those that were seen by both screeners and those that were only seen by one of the screeners. I would then create a completely new project in ASReview with the exact same classifier & data extraction technique as before, and upload this merged Excel file in this new project. 

This way, all screening that has been done until now would serve as prior knowledge, and given that we use the same model configuration (i.e., classifier + feature extractor), _the screening should just continue where it was left off_ . 
Thus, this approach seemed to me to be the most straightforward way of moving from double-screening to single-screening without losing information - is that correct or am I overlooking something?
Or perhaps there is also a simpler way of doing this than manually adding the labels?

Second, I was wondering to what extent this would affect the reproducibility of the review. In my view, as long as we make all the project files available, it would not negatively impact the review's reproducibility. 

Looking forward to reading your thoughts on this!
Best wishes!

**Comment by bethgr** [comment]
An idea regarding multiple screeners: Sam Langton created a helpful ASReview report tool for inter-rater reliability. You can view an example of the report here: https://rpubs.com/langton_/asreview_report_example. Links to the repository and instructions can also be found on Sam's page.
